src/nfv_test_api/util.py

```diff
@@ -14,7 +14,7 @@
 def run_in_ns(namespace: str, command: List[str]) -> str:
     """ Run the given command in the given namespace and return the result as a string
     """
-    output = subprocess.check_output(["ip", "netns", "exec", namespace] + command)
+    output = subprocess.check_output(["ip", "netns", "exec", namespace] + command, stderr=subprocess.PIPE)
```

Comment:
According to the documentation `subprocess.STDOUT` should be used to also capture standard error.

Reply:
You're right, however, as I understand it they mean you should use `subprocess.STDOUT` if you want to capture stderr in the return value of `subprocess.check_output` (redirect stderr to stdout, wich gets captured and returned). This is not what we want: we want stderr to be available on the `subprocess.CalledProcessError` object. This achieves that by opening a pipe to the standard stream (https://docs.python.org/3.8/library/subprocess.html#subprocess.PIPE).
I've tried both (because my initial interpretation was the same as yours). The error log on `main.py:244` puts the error under `out` when using `subprocess.STDOUT` and under `err` when using `subprocess.PIPE`.

Reply:
Ok. I got it.